package.json

```diff
@@ -72,7 +72,7 @@
   "dependencies": {
     "@bugsnag/js": "7.20.2",
     "@fastify/static": "6.10.2",
-    "@netlify/blobs": "6.5.0",
+    "@netlify/blobs": "^7.0.0",
```

Comment:
why is this the only dependency with a caret?

Reply:
Good catch! We're pinning the exact versions of dependencies to avoid issues with transitive dependencies. It shouldn't be an issue here, since `@netlify/blobs` doesn't have any dependencies, but I've pinned it anyway in https://github.com/netlify/cli/pull/6418/commits/b0572b1a55f1ef38016480a339ddc32d2c038076.